airbyte-server/src/test/java/io/airbyte/server/handlers/SourceHandlerTest.java

```diff
@@ -139,6 +139,39 @@ void testCreateSource() throws JsonValidationException, ConfigNotFoundException,
     verify(validator).ensure(sourceDefinitionSpecificationRead.getConnectionSpecification(), sourceConnection.getConfiguration());
   }
 
+  @Test
+  void testUpdateSourceName() throws JsonValidationException, ConfigNotFoundException, IOException {
```

Comment:
instead of a separate test, what if we combine this into the existing testUpdateSource test?